FF1Blazorizer/Tabs/MapsTab.razor

```diff
@@ -31,13 +31,29 @@
 	</div>
 	<div class="col2">
 		<h4>Overworld</h4>
-		<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="mapOpenProgressionCheckBox" @bind-Value="Flags.MapOpenProgression">Early Open Progression</TriStateCheckBox>
-		<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="mapOpenProgressionDwavesNWCheckBox" Indent IsEnabled="@Flags.MapOpenProgression" @bind-Value="Flags.MapDwarvesNorthwest">Extended Open Progression</TriStateCheckBox>
-		<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="mapOpenProgressionDocksCheckBox" @bind-Value="Flags.MapOpenProgressionDocks">Northern Docks</TriStateCheckBox>
-		<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="mapOpenProgressionFloaterDockCheckBox" IsEnabled="@Flags.MapOpenProgression" @bind-Value="Flags.MapAirshipDock">Ryukahn Desert Dock</TriStateCheckBox>
-		<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="bahamutCardiaDockCheckBox" @bind-Value="Flags.MapBahamutCardiaDock">Bahamut Cardia Dock</TriStateCheckBox>
-		<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="lefeinRiverDockCheckBox" @bind-Value="Flags.MapLefeinRiver">Lefein River Dock</TriStateCheckBox>
-		<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="gaiaMountainPassCheckBox" @bind-Value="Flags.MapGaiaMountainPass">Gaia Mountain Pass</TriStateCheckBox>
+		<TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="mapOpenProgressionCheckBox"
+		IsEnabled="@(Flags.OwMapExchange == OwMapExchanges.None)" @bind-Value="Flags.MapOpenProgression">Early Open Progression</TriStateCheckBox>

```

Comment:
I'm pretty sure map changes are also compatible with the 3 alternative start maps?

Reply:
It's ugly but I tweaked it so those are accepted too.